modules/serverpod_auth/serverpod_auth_shared_flutter/lib/src/image_uploader.dart

```diff
@@ -69,16 +69,17 @@ class ImageUploader {
         maxHeight: imageSize,
         maxWidth: imageSize,
         aspectRatio: const CropAspectRatio(ratioX: 1.0, ratioY: 1.0),
-        cropStyle: CropStyle.circle,
         uiSettings: [
           AndroidUiSettings(
               toolbarTitle: 'Cropper',
               toolbarColor: toolbarColor,
               toolbarWidgetColor: toolbarWidgetColor,
               initAspectRatio: CropAspectRatioPreset.original,
+              cropStyle: CropStyle.circle,
               lockAspectRatio: false),
           IOSUiSettings(
             minimumAspectRatio: 1.0,
+            cropStyle: CropStyle.circle,
```

Comment:
Suggestion: It seems like this change was introduced in version `7.0.0` of image cropper. Could we add a range instead, starting from that version? That is `'>=7.0.0 <9.0.0'`

This would allow us to keep our lowest dart version and probably fix the issues encountered in in the CI.

Reply:
Yeah that is better. I have updated the version.